sympy/integrals/rubi/utility_function.py

```diff
@@ -444,7 +444,7 @@ def ArcCosh(a):
     return acosh(a)
 
 class Util_Coefficient(Function):
-    def doit(self):
+    def doit(self, **hints):
```

Comment:
It would be better if this was change to `_eval_doit` rather than modifying the signature of `doit`.

Reply:
Got it.